### PR TITLE
Virtualization SDK Docs - New plugin config property snapshotParametersDefinition

### PR DESCRIPTION
**Pull request by jeffngo**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build was run locally and any changes were pushed
- [ ] Lint has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Added `feature` to `mkdocs.yml` to get past an error I was seeing trying to run `pipenv run mkdocs serve`.
```
  File "/Users/jngo/Documents/workspace/vsdk/docs/material/404.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/Users/jngo/Documents/workspace/vsdk/docs/material/base.html", line 118, in top-level template code
    {% if feature.tabs %}
  File "/Users/jngo/virtualenv/mkdocs/lib/python3.9/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'mkdocs.theme.Theme object' has no attribute 'feature'
```
- Included the fix that I merged with trunk (https://github.com/delphix/virtualization-sdk/commit/707d0de42e843d2f8673d6e27df035e734387bba)

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/docs/References/Schemas.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Delphix Object | Schema | Autogenerated Class
 [Linked Source](Glossary.md#linked-source) | [LinkedSourceDefinition](Schemas_and_Autogenerated_Classes.md#linkedsourcedefinition-schema) | [LinkedSourceDefinition](Schemas_and_Autogenerated_Classes.md#linkedsourcedefinition-class)
 [Virtual Source](Glossary.md#virtual-source) | [VirtualSourceDefinition](Schemas_and_Autogenerated_Classes.md#virtualsourcedefinition-schema)| [VirtualSourceDefinition](Schemas_and_Autogenerated_Classes.md#virtualsourcedefinition-class)
 [Snapshot](Glossary.md#linked-source) | [SnapshotDefinition](Schemas_and_Autogenerated_Classes.md#snapshotdefinition-schema) | [SnapshotDefinition](Schemas_and_Autogenerated_Classes.md#snapshotdefinition-class)
+[Snapshot Parameter](Glossary.md#snapshot-parameter) | [SnapshotParametersDefinition](Schemas_and_Autogenerated_Classes.md#snapshotparametersdefinition-schema) | [SnapshotParametersDefinition](Schemas_and_Autogenerated_Classes.md#snapshotparametersdefinition-class)
```
Comment (ravi-cm) [inline docs/docs/References/Schemas.md:25]: Snapshot Parameters?

Comment (ravi-cm) [inline docs/docs/References/Schemas.md:25]: snapshot-parameters?

File: docs/docs/References/Schemas_and_Autogenerated_Classes.md [hunk 2]
```diff
@@ -230,3 +230,40 @@ snapshot2.transaction_id = 1500
 # A snapshot that omits the optional "transaction_id" property
 snapshot3 = SnapshotDefinition(version="1.0.0")
 ```
+
+## SnapshotParametersDefinition
+
+Defines [Snapshot Parameters](Glossary.md#snapshot-parameter) for the snapshot operation.
```
Comment (ravi-cm) [inline docs/docs/References/Schemas_and_Autogenerated_Classes.md:236]: snapshot-parameters?

File: docs/docs/References/Glossary.md [hunk 3]
```diff
@@ -88,10 +88,10 @@ Information that represents a set of dependencies that a dataset requires in ord
 A formal description of a data type. Plugins use JSON format for their [schemas](Schemas_and_Autogenerated_Classes.md#schemas-and-autogenerated-classes).
 
 ## Snapshot
-A point-in-time read-only copy of a dataset. A snapshot includes associated metadata represented by the [SnapshotDefinition Schema](Schemas_and_Autogenerated_Classes.md#snapshotdefinition)
+A point-in-time read-only copy of a dataset. A snapshot includes associated metadata represented by the [SnapshotDefinition Schema](Schemas_and_Autogenerated_Classes.md#snapshotdefinition).
 
 ## Snapshot Parameter
```
Comment (ravi-cm) [inline docs/docs/References/Glossary.md:93]: Snapshot Parameters?

**Review by ravi-cm — APPROVED** [review]
LGTM.

**Review by ravi-cm — APPROVED** [review]

**Review by nhlien93 — APPROVED** [review]

**Pull request merged**